### DISCUSSION TITLE
Virtualization is not supported on your system. Macbook Air M1 Monterey 12.3.1

### DISCUSSION THREAD

**Discussion by my-cla4ik-f0rmy** [utmapp/UTM#3944] [UTM Help (macOS)]
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/89280222/165255511-aa9d20ab-0330-4268-abef-d9e3b1452967.png">

**Comment by osy** [comment]
Make sure you’re not running it in Rosetta.

**Reply by my-cla4ik-f0rmy** [reply]
Oh yes, Rosetta... 
Thank you!
